scripts/run_performance_job.py

```diff
@@ -729,18 +729,14 @@ def run_performance_job(args: RunPerformanceJobArgs):
                 os.path.join(args.performance_repo_dir, "src", "tools", "PerfLabGenericEventSourceLTTngProvider", "build.sh"),
                 "-o", os.path.join(payload_dir, "PerfLabGenericEventSourceForwarder")]).run()
         
-        # download PDN
-        if args.os_group == "windows" and args.architecture != "x86" and args.download_pdn:
-            print("Downloading PDN")
-            escaped_upload_token = str(os.environ.get("PerfCommandUploadTokenLinux")).replace("%25", "%")
-            pdn_url = f"https://pvscmdupload.blob.core.windows.net/assets/paint.net.5.0.3.portable.{args.architecture}.zip{escaped_upload_token}"
+        # copy PDN
+        if args.os_group == "windows" and args.architecture != "x86" and args.pdn_path is not None:
```

Comment:
Is the reasoning behind this for when people do local runs? We shouldn't need this check as the yaml ensures that pdn_path is empty unless we are in the right OS/Arch combination.

Reply:
Yes, the check is just for local runs